lib/platform/browser.dart

```diff
@@ -111,11 +100,21 @@ PlatformRef browserPlatform() {
 /// multiple applications on a page, Angular treats each application injector's
 /// services as private to that application.
```

Comment:
Not anymore (we did, in the doc examples, but there were issues so I worked around them), ... at least not that I am aware of.